src/components/App.css

```diff
@@ -3,3 +3,7 @@ body {
   padding: 0;
   font-family: sans-serif;
 }
+
+.App-intro {
+  font-size: large;
```

Comment:
Adding this at a random line, but can we move all tests to a test/ directory? Also, how are you testing currently? Can we use ‘mocha’?

Reply:
We're currently using jest becomes it comes with create-react-app, they want to use their testing library :)

But we can easily eject and use whatever tools we want. Why do you prefer Mocha?

I'm not against using a tests/ directory in the root. But I also like the idea of keeping unit tests close to the source code they are testing. Something like this:

../someDirectory/someService.js
../SomeDirectory/tests/someService.test.js

What do you think about this approach?

Reply:
+1 to keeping tests close to source code especially since we are doing three different projects in one repo (front-end web app, back-end web app, and solidity contracts).

Are there any major differences between `mocha` and `jest`? Only reason why I suggested mocha is that mocha is the big one in the ethereum ecosystem.